src/GitVersionHelper.php

```diff
@@ -39,7 +41,10 @@ public static function getVersion()
         chdir(base_path());
 
         // Get version string from git
-        $output = shell_exec('git describe --always --tags --dirty');
+        $command = 'git describe --always --tags --dirty';
+        $process = new Process($command);
```

Comment:
Let's put this all in one line, and pass the command as an array as recommended here: https://symfony.com/doc/current/components/process.html#using-features-from-the-os-shell